lib/src/rules/directives_ordering.dart

```diff
@@ -199,6 +247,31 @@ class _Visitor extends SimpleAstVisitor {
         .forEach(reportDirective);
   }
 
+  void _checkDirectiveSectionOrderedAlphabetically(
+      Set<AstNode> lintedNodes, CompilationUnit node) {
+    final importDirectives = _getImportDirectives(node);
+    final exportDirectives = _getExportDirectives(node);
+
+    final dartImports = importDirectives.where(_isDartImport);
+    final relativeImports = importDirectives.where(_isRelativeImport);
+
+    _checkSectionInOrder(lintedNodes, dartImports);
+    _checkSectionInOrder(lintedNodes, relativeImports);
+    _checkSectionInOrder(lintedNodes, exportDirectives);
+
+    if (project != null) {
```

Comment:
I copied from another lint who has this issue attached https://github.com/dart-lang/linter/issues/154